BE/src/main/java/com/sidedish/domain/Item.java

```diff
@@ -40,12 +44,21 @@ public Item(String title, String description, BigDecimal price, double discountR
         this.quantity = quantity;
         this.rewardPoint = rewardPoint;
         this.images = images;
+        this.shipInfo = shipInfo;
     }
 
     void addOrders(Order order) {
         this.orders.add(new OrderRef(order.getId()));
     }
 
+    public void decreasedQuantity(int quantity) {
+        this.quantity -= quantity;
+    }
+
+    public boolean isValidQuantity(int quantity) {
+        return this.quantity - quantity > 0;
```

Comment:
스타일 차이지만, 요렇게 가도 되겠네요.

```suggestion
        return this.quantity > quantity;
```